lib/index.js

```diff
@@ -240,9 +240,10 @@ class Service {
 
     // Force the {raw: false} option as the instance is needed to properly
     // update
-    const updateOptions = Object.assign({ raw: false }, params.sequelize);
+    const updateOptions = Object.assign({}, params.sequelize, { raw: false });
```

Comment:
I believe you want to reverse the order of this `Object.assign()`. Otherwise `raw` will always be set to false? I see the comment above "as the instance is needed to properly update", but I also see that the user can override it with with the old code.